io_scene_dae/export_dae.py

```diff
@@ -556,7 +542,7 @@ def export_mesh(self, node, armature=None, skeyindex=-1, skel_source=None,
         if armature is not None:
             si = self.skeleton_info[armature]
 
-        has_tangents = self.config["use_tangent_arrays"]  # could detect..
+        has_tangents = self.config["use_tangent_arrays"]  # TODO: Review
```

Comment:
@akien-mga Did not read your comments above, I have implemented the change you suggested.